packages/next/test/fixtures/00-app-dir-catchall-pages/probes.json

```diff
@@ -12,7 +12,7 @@
     },
     {
       "path": "/en/about",
-      "status": 200,
+      "status": 404,
```

Comment:
Is this expected to be 404 now? <picture data-single-emoji=":thonking2:" title=":thonking2:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thonking2/63d309b4d143d783.gif" alt=":thonking2:" align="absmiddle"></picture>

Reply:
Trying an approach in [e8e5e04](https://github.com/vercel/vercel/pull/10415/commits/e8e5e049bc1036574152c07c26b6f1167ce99220) to avoid the 404s 🤞